content/zh/news/releases/1.12.x/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
It sense that you deleted old content.

Reply:
Yeah, maybe I didn't notice. It's been corrected

Reply:
you didn't fix this.